linux/riscv64.patch

```diff
@@ -1,87 +0,0 @@
-diff --git PKGBUILD PKGBUILD
-index 2b0f6d9..b1b9999 100644
---- PKGBUILD
-+++ PKGBUILD
-@@ -31,6 +31,8 @@ source=(
-   https://cdn.kernel.org/pub/linux/kernel/v${pkgver%%.*}.x/${_srcname}.tar.{xz,sign}
-   $url/releases/download/$_srctag/linux-$_srctag.patch.zst{,.sig}
-   config  # the main kernel config file
-+  riscv64.config-patch
```

Comment:
I assume the other changes are still needed...

Reply:
done